packages/react-ui/src/widgets/legend/LegendCategories.js

```diff
@@ -65,61 +64,70 @@ LegendCategories.propTypes = {
 
 export default LegendCategories;
 
-// Aux
-const useStyles = makeStyles((theme) => ({
-  legendCategories: {
-    alignItems: 'center'
-  },
-  marker: {
-    whiteSpace: 'nowrap',
-    display: 'block',
-    width: '12px',
-    height: '12px',
+const getCircleStyles = ({ isMax, color, isStrokeColor, theme }) => ({
+  border: '2px solid transparent',
+  '&::after': {
+    position: 'absolute',
+    display: isMax ? 'block' : 'none',
+    content: '""',
+    width: '16px',
+    height: '16px',
```

Comment:
Please, use `theme.spacing()` for size values like this

Reply:
Agree, and then we can merge